q-pup_config/include/quadruped_description.h

```diff
@@ -0,0 +1,34 @@
+#ifndef QUADRUPED_DESCRIPTION_H
+#define QUADRUPED_DESCRIPTION_H
+
+#include <quadruped_base/quadruped_base.h>
+
+namespace champ
+{
+    namespace URDF
+    {
+        void loadFromHeader(champ::QuadrupedBase &base)
+        {
+      base.lf.hip.setOrigin(0.3149, 0.2384, 0.0, 0.0, 0.0, 0.0);
+base.lf.upper_leg.setOrigin(0.0906, 0.0815, -0.0, 0.0, 0.0, 0.0);
+base.lf.lower_leg.setOrigin(0.0, 0.0, -0.4, 0.0, 0.0, 0.0);
+     base.lf.foot.setOrigin(0.0, 0.0, -0.3, 0.0, 0.0, 0.0);
```

Comment:
```suggestion
     base.lf.foot.setOrigin(0.0, 0.0, -0.35, 0.0, 0.0, 0.0);
```

Tool should be rerun to update this everywhere